examples/shader/shader.wgsl

```diff
@@ -0,0 +1,42 @@
+// STAGE: VERTEX ---------------------------------------------------------------------------------
+struct VertexOutput {
+    [[builtin(position)]] position: vec4<f32>;
+    [[location(0)]] local_position: vec3<f32>;
+};
+
+struct Renderer {
+    proj_view: mat4x4<f32>;
+};
+[[group(0), binding(0)]]
+var<uniform> u_renderer: Renderer;
+
+[[stage(vertex)]]
+fn vs_main(
+    [[location(0)]] position: vec3<f32>,
+) -> VertexOutput {
+    var out: VertexOutput;
+    out.local_position = position;
+    out.position = u_renderer.proj_view * vec4<f32>(position, 1.0);
+    return out;
+}
+
+// STAGE: FRAGMENT -------------------------------------------------------------------------------
+struct Gradient {
+  zenith_color: vec4<f32>;
+  nadir_color: vec4<f32>;
+};
+[[group(0), binding(1)]]
+var<uniform> u_skybox: Gradient;
```

Comment:
Most likely the variable name must be `u_gradient`

Reply:
Done